gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/util/ResourceFilterUtilTest.java

```diff
@@ -157,6 +162,26 @@ private static Stream<Arguments> nonSpecimenTestFiles() {
         );
     }
 
+    @Test
+    public void testIsReferralRequestToExternalDocumentLinkSet() {
```

Comment:
Been a bit distracted, but my initial thought @MartinWheelerMT is that the following implementation
```java
public static bool isReferralRequestToExternalDocumentLinkSet() {
  return true;
}
```

Would pass the tests, so I'm not sure why your prod code is so complicated.